api/db/changelog/db.changelog-154-wgs-cromwell-extract-submission.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="calbach" id="changelog-154-wgs-cromwell-extract-submission">
+
+    <createTable tableName="wgs_extract_cromwell_submission">
+      <column name="wgs_extract_cromwell_submission_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="submission_id" type="VARCHAR(80)">
+        <constraints unique="true" nullable="false"/>
+      </column>
+      <column name="workspace_id" type="bigint">
+        <constraints nullable="false" />
+      </column>
+      <column name="creator_id" type="bigint" />
+      <column name="data_set_id" type="bigint" />
+      <column name="sample_count" type="bigint" />
+      <column name="creation_time" type="datetime">
```

Comment:
FYI: I mistakenly used "timestamp" here - fixed that to datetime and supplied in the service. I also fixed sample_count to be nullable and set to NULL for now.